### PR TITLE
added ffmpeg to dockerfile and fixed test

### PR DESCRIPTION
**Pull request by kashif**

### CODE REVIEW

File: gym/monitoring/tests/test_video_recorder.py [hunk 1]
```diff
@@ -22,16 +22,17 @@ def render(self, mode=None):
         pass
 
 # TODO(jonas): disabled until we have ffmpeg on travis
```
Comment (jonasschneider) [inline gym/monitoring/tests/test_video_recorder.py]: Great! Feel free to also remove this note.

**Comment by kashif** [comment]
@jonasschneider will do... i wanted to just see the docker/ubuntu error... i will fix that first

**Comment by jonasschneider** [comment]
Thanks for the patch. Instead of using the third-party `ffmpeg`, could you rather install `libav-tools` to have `avconv` (which we also support in `VideoRecorder`)? I'd rather not have the unofficial PPA here, if we have the alternative.

**Comment by jonasschneider** [comment]
Awesome, thank you!

**Pull request merged**